packages/client/src/views/SysAdmin/Team/user/userCreation/CreateNewUser.tsx

```diff
@@ -275,8 +264,9 @@ const mapStateToProps = (state: IStoreState, props: Props) => {
   }
 }
 
-export const CreateNewUser = connect(mapStateToProps, {
-  goBack,
-  clearUserFormData,
-  fetchAndStoreUserData
-})(injectIntl(withApollo<Props>(CreateNewUserComponent)))
+export const CreateNewUser = withRouter(
+  connect(mapStateToProps, {
+    clearUserFormData,
+    fetchAndStoreUserData
+  })(injectIntl(withApollo<Props>(CreateNewUserComponent)))
+)
```

Comment:
We should change it back to functional component

Reply:
Added comment